app/models/move.rb

```diff
@@ -126,7 +126,9 @@ def rebooked
   end
 
   def rebook
-    rebooked || self.class.create(
+    return rebooked if rebooked.present?
+
+    Move.create!(
```

Comment:
just a minor point, but surprised rubocop hasn't flagged a preference for `self.class.create!`

Reply:
Feel free to ignore this comment if you like 😀